README.md

```diff
@@ -38,7 +36,7 @@ Rubygems:
 
 Manually:
 
-    git clone git@github.com:darrenli/jira-cli.git
+    git clone git@github.com:ruby-jira/jira-cli.git
     cd jira-cli
     ./scripts/install
 
```

Comment:
The Project Installation section below needs updating :smile: 

Maybe consider removing the disclaimer, too - since we are aiming for `v1.0.0`, hopefully it's fairly stable by now!